plugins/woocommerce/tests/e2e-pw/utils/login.js

```diff
@@ -1,7 +1,8 @@
 const { expect } = require( '../fixtures/fixtures' );
 const logIn = async ( page, username, password, assertSuccess = true ) => {
-	await page.waitForSelector( 'form' );
+	await page.getByLabel( 'Username or Email Address' ).click();
```

Comment:
Meant to add a comment on this:

I know this is a goofy thing, and a `.fill()` shouldn't need a `.click()` preceding it.  It looks like in about 10% of executions, Playwright starts to fill in the login form before the page has finished loading and that there is a `focus()` on the username field that fires right before we start filling out the password -- so the password ends up overwriting the username (and that causes the flakiness/failure).  Explicitly clicking on each field before filling it out seems to have fixed that.